gofor/test.js

```diff
@@ -1,35 +1,35 @@
 const Gofor = require('./');
 
-const defaults = {
+const defaults = () => ({
```

Comment:
You're returning the object so no one can mutate it?
if so, why not freezing it?

Reply:
The constructor is allowed to mutate it. I just want a new copy for each test